src/components/Dropdown/Dropdown.tsx

```diff
@@ -77,10 +90,11 @@ export const Dropdown = ({
 
   const menu = useMemo(() => ({
     items: antdItems,
-    onClick: onAntdMenuClick,
     /* istanbul ignore next */
     getPopupContainer: (triggerNode: HTMLElement) => (document.querySelector(`.${uniqueClassName}`) || triggerNode) as HTMLElement,
-  }), [antdItems, onAntdMenuClick, uniqueClassName])
+    onClick: onAntdMenuClick,
+    selectedKeys: selectedItems,
```

Comment:
does it make sense to check here or somewhere else whether multiple `selectedItems` have been passed with `multiple: false`, and in case only take the first or throw an error?

Reply:
there is no need for doing that here, we could make such assertion at first render; however what would be the point? we could:

- throw an error: dangerous at runtime
- ignore the provided `intialSelectedItems`: it would end up in a weird interface
- log a console.warn|error: possibly the best choice, however we have no usecase yet of such behaviours
- do nothing and accept and undefined behaviour (which is the strategy applied here)

At the end of the day is a simple misconfiguration, I'd go with the the last option.

Reply:
Got the point, yeah I agree with you we can just ignore it